tests/db_functions/math/test_round.py

```diff
@@ -38,9 +38,9 @@ def test_integer(self):
             normal_round=Round('normal'),
             big_round=Round('big'),
         ).first()
-        self.assertIsInstance(obj.small_round, int)
-        self.assertIsInstance(obj.normal_round, int)
-        self.assertIsInstance(obj.big_round, int)
+        self.assertIsInstance(obj.small_round, float)
+        self.assertIsInstance(obj.normal_round, float)
+        self.assertIsInstance(obj.big_round, float)
```

Comment:
We should fix this side-effect, probably by providing a custom `Round._resolve_output_field()` method, .e.g.
```python
class Round(FixDecimalInputMixin, Transform):
    ...

    def _resolve_output_field(self):
        source = self.get_source_expressions()[0]
        return source.output_field
```